app/controllers/reactions_controller.rb

```diff
@@ -63,14 +63,12 @@ def create
         reactable_type: params[:reactable_type],
         category: category,
       )
-      if reaction.save
-        result = "create"
-        Moderator::SinkArticles.call(reaction.user_id) if vomit_reaction_on_user?(reaction)
-        Notification.send_reaction_notification(reaction, reaction.reactable.user)
-        Notification.send_reaction_notification(reaction, reaction.reactable.organization) if organization_article?(reaction)
-      else
-        return render json: { error: reaction.errors.full_messages.join(", "), status: 422 }, status: :unprocessable_entity
-      end
+      return render json: { error: reaction.errors.full_messages.join(", "), status: 422 }, status: :unprocessable_entity unless reaction.save
```

Comment:
Personally I'm not a big fan of modifier `if`/`unless` on long lines, it's easy to overlook. This seems clearer to me:

```ruby
unless reaction.save
  render json: { error: reaction.errors.full_messages.join(", "), status: 422 }, status: :unprocessable_entity
  return
end
```
Just a suggestion though, don't let it stop you 😃

Reply:
Also: Rails has a [`to_sentence`](http://api.rubyonrails.org/v5.2.4/classes/Array.html#method-i-to_sentence) method that comes in handy in cases like this:

```ruby
reaction.errors.full_messages.to_sentence
```

Reply:
TIL about `to_sentence`!

Reply:
Rubocop really wanted this to be a guard clause, but I'll see if I can make it readable.

Reply:
@jacobherrington The trick is to put two statements into the `unless`, then it shouldn't suggest to make it a guard 😉 Unless it got smarter over the years, I actually haven't tried that.